### PR TITLE
Update Readme for supported JDK versions and 22005 NOTICES

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

File: dev/build.image/publish/README.TXT [hunk 1]
```diff
@@ -35,8 +35,8 @@ If a serverName is not specified, "defaultServer" will be used.
 
 Notes:
 
-The server requires Java 8.   To download IBM Java SDKs, visit 
-http://www.ibm.com/developerworks/java/jdk/ 
+The server requires Java 8, 11 or 17   To download IBM Java SDKs, visit
```
Comment (dmuelle) [inline dev/build.image/publish/README.TXT]:
```suggestion
The server requires Java 8, 11 or 17. To download IBM Java SDKs, visit 
```

**Comment by skasund** [comment]
L2 message review not required

**Review by hlhoots — APPROVED** [review]
LGTM

**Comment by cbridgha** [comment]
#libby

**Pull request merged**